charts/prometheus-nats-exporter/values.yaml

```
@@ -6,7 +6,6 @@ replicaCount: 1
 
 image:
   repository: natsio/prometheus-nats-exporter
```

I would keep the option but let it be empty.

```suggestion
  repository: natsio/prometheus-nats-exporter
   tag: ""
```